dashboard/test/ui/features/signingIn.feature

```diff
@@ -56,12 +56,15 @@ Scenario: Join non-existent section from sign in page shows error
   And I type "9999999999" into "#section_code"
   And I click selector "#section_form .btn-primary"
   Then I wait until I am on "http://studio.code.org/courses"
+  Then I wait to see ".alert-danger"
   And element ".alert-danger" contains text "Could not find a section with code"
 
-@as_taught_student @no_ie @no_safari
+@as_taught_student
 Scenario: Join existing section from sign in page goes to section join page
   Given I sign out
   Given I am on "http://studio.code.org/users/sign_in/"
   And I type the section code into "#section_code"
   And I click selector "#section_form .btn-primary"
+  Then I wait until current URL contains "http://studio.code.org/join"
```

Comment:
Again, great.  I think [you could also](https://github.com/code-dot-org/code-dot-org/blob/87a467b23bfe8f44e31682df9a1801ca55b416ae/dashboard/test/ui/features/step_definitions/steps.rb#L255-L268)

```gherkin
And I press the first "#section_form .btn-primary" element to load a new page
And element ".main" eventually contains text "Register to join the class"
```